net/chrony/files/chronyd.init

```diff
@@ -21,12 +22,30 @@ handle_source() {
 	config_get maxpoll "$cfg" maxpoll
 	config_get_bool iburst "$cfg" iburst 0
 	config_get_bool nts "$cfg" nts 0
+	config_get_bool prefer "$cfg" prefer 0
+	config_get_bool xleave "$cfg" xleave 0
+	config_get maxdelay "$cfg" maxdelay
+	config_get mindelay "$cfg" mindelay
+	config_get maxsamples "$cfg" maxsamples
+	config_get minsamples "$cfg" minsamples
+	config_get port "$cfg" port
+	config_get ntsport "$cfg" ntsport
+	config_get maxsources "$cfg" maxsources
 	echo $(
 		echo $sourcetype $hostname
 		[ -n "$minpoll" ] && echo minpoll $minpoll
 		[ -n "$maxpoll" ] && echo maxpoll $maxpoll
 		[ "$iburst" = "1" ] && echo iburst
 		[ "$nts" = "1" ] && echo nts
+		[ "$prefer" = "1" ] && echo prefer
+		[ "$xleave" = "1" ] && [ "$cfg" != "pool" ] && echo xleave
```

Comment:
Why is xleave limited to non-pool sources?

Reply:
It's my reading from the man page that it functions only in those...? Is that not correct?